### DISCUSSION TITLE
Scalability and resource usage vs Soketi

### DISCUSSION THREAD

**Discussion by binaryfire** [dunglas/mercure#855] [Q&A] [answered]
Hi all. How does a single Mercure instance compare to Soketi in terms of concurrent connections and resource usage? I've used Soketi with Laravel in the past and it's been able to handle a crazy number of connections on a fairly small VPS. I'd like to try Mercure but just want to make sure it will perform similarly.

**Comment by dunglas** [comment, marked as answer]
Hi,

Here are some insights:

- [Pushing 8 millions of Mercure notifications per day to run mail.tm](https://les-tilleuls.coop/en/blog/mail-tm-mercure-rocks-and-api-platform)
- [How Raven Controls uses Mercure to power big events such as Cop 21 and Euro 2020](https://api-platform.com/con/2022/conferences/real-time-and-beyond-with-mercure/)
- [100,000 simultaneous Mercure users to power iGraal](https://speakerdeck.com/dunglas/mercure-real-time-for-php-made-easy?slide=52)

**Reply by binaryfire** [reply]
@dunglas Thanks. Impressive numbers 👌
